src/app/input_gen.ml

```diff
@@ -0,0 +1,60 @@
+open Biokepi.EDSL.Library.Input
+open Nonstd
+module String = Sosa.Native_string
+
+let cmd () =
+  let open Cmdliner in
+  let version = "0.0.0" in
+  let doc = "Submit a PGV Pipeline" in
+  let man = [
+    `S "Description";
+    `P "The PGV Pipeline takes FASTQs and runs them through the PGV pipeline.";
+  ] in
+  let name' =
+    let doc = "Name of the dataset; should be unique."
+    in
+    Arg.(required & opt (some string) None & info ["name"; "N"] ~doc)
+  in
+  let host =
+    let doc = "Host where the files are stored, e.g. ssh://dev1,\
+               assuming there is a named SSH host at dev1."
+    in
+    Arg.(required & opt (some string) None & info ["host"; "H"] ~doc)
+  in
+  let fastqs_directory =
+    let doc = "Directory on `host` where tumor fastq.gzs can be found." in
+    Arg.(required & opt (some string) None & info ["directory"; "d"] ~doc)
+  in
+  Term.(pure (fun name host directory ->
+      let host = match Ketrew_pure.Host.of_string host with
+      | `Ok host -> host
+      | `Error msg -> printf "Error parsing host.\n%!"; exit 1
+      in
+      let open Pvem_lwt_unix.Deferred_result in
+      Derive.fastqs ~host:host directory
```

Comment:
Having the option to pass false to `~paired_end` from the CLI would be nice also